open-banking-accelerator/components/com.wso2.openbanking.accelerator.identity/src/main/java/com/wso2/openbanking/accelerator/identity/app2app/cache/JTICache.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.wso2.openbanking.accelerator.identity.app2app.cache;
+
+import com.wso2.openbanking.accelerator.identity.cache.IdentityCache;
+import com.wso2.openbanking.accelerator.identity.cache.IdentityCacheKey;
+
+/**
+ * Class for maintaining JTI cache.
+ */
+public class JTICache {
+
+    private static volatile IdentityCache jtiCacheInstance;
+
+    /**
+     * Get JTI cache instance.
+     *
+     * @return IdentityCache instance as JTICache
+     */
+    public static IdentityCache getInstance() {
+
+        //Outer null check avoids entering synchronized block when jtiCache is not null.
+        if (jtiCacheInstance == null) {
+            // Synchronize access to ensure thread safety
+            synchronized (JTICache.class) {
+                // Avoids race condition withing threads
```

Comment:
typo

Reply:
Resolved in [3180209](https://github.com/wso2/financial-open-banking/pull/67/commits/31802097d4d29819a9fa74b7e2626e2f0e659b0c).